cohortextractor/tpp_backend.py

```diff
@@ -383,15 +412,37 @@ def get_queries(self, covariate_definitions):
             for name in table_queries
             if name != "population"
         ]
+
+        wheres = [f'{output_columns["population"]} = 1']
+
+        def get_t1oo_exclude_expressions():
+            # If this query has been explictly flagged as including T1OO patients then
+            # return unmodified
+            if self.include_t1oo:
+                return [], []
+            # Otherwise we add an extra LEFT OUTER JOIN on the T1OO table and
+            # WHERE clause which will exclude any patient IDs found in the T1OO table
+            return (
+                [
+                    f"LEFT OUTER JOIN {T1OO_TABLE} ON {T1OO_TABLE}.Patient_ID = {patient_id_expr}"
+                ],
+                ["{T1OO_TABLE}.Patient_ID IS null"],
```

Comment:
```suggestion
                [f"{T1OO_TABLE}.Patient_ID IS null"],
```
If I had a penny for every time I'd forgotten to add the `f` prefix ...